spec/features/items_view_spec.rb

```diff
@@ -46,7 +47,7 @@
     click_link_or_button("Edit Item")
     fill_in('item[title]', with: "Racoon Ragu" )
     fill_in('item[description]', with: "A bandit of a meal" )
-    fill_in('item[price]', with: 5.34)    
+    fill_in('item[price]', with: 5.34)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.